src/components/ReviewFormContainer/index.tsx

```diff
@@ -102,23 +102,18 @@ function ReviewFormContainer() {
    * submitForm
    * Update the form values again but this time submit to the API.
    */
-  const submitForm = (formData) => {
+  const submitForm = () => {
```

Comment:
Why doesn't this need to take an argument anymore?

Reply:
For this component, there are two submit functions `editSectionInfo` and `submitForm`. The former updates individual forms in each section but the latter submits the entire form and makes the API call. Both of those are part of and using `react-hook-form` to get form submission values.

In each section, `editSectionInfo` gets the `formData` for a specific section and updates the app's form submission value (in `formValues`). In `submitForm`, no new data is being updated and all that needs to be done is submit the app's form submission values. So that's why `formData` can be removed as a param since we don't need it from `react-hook-form`.